testing-tools/Dockerfile

```diff
@@ -43,6 +44,11 @@ RUN apt-get update && \
 COPY testing-tools/python /stackable/python
 COPY testing-tools/licenses /licenses
 
+ENV PATH=/stackable/keycloak/bin:$PATH
+# todo: do we need to maintain this via the mirror script?
+RUN curl --fail https://repo.stackable.tech/repository/packages/keycloak/keycloak-${KEYCLOAK_VERSION}.tar.gz | tar -xzC /stackable && \
```

Comment:
@lfrancke we have checked the sha1 hash and uploaded [keycloak-23.0.0.tar.gz](https://github.com/keycloak/keycloak/releases/download/23.0.0/keycloak-23.0.0.tar.gz) manually.

Should we add a script to take care of this like what it done for [hbase](https://github.com/stackabletech/docker-images/blob/89e39e9db8e06846dc825076c2a74c55210298a6/hbase/upload_new_hbase-operator-tools_version.sh)?

Reply:
I think we'll leave it for now and it can be done when the Keycloak dependency is updated.